balancer/weightedroundrobin/balancer_test.go

```diff
@@ -87,7 +88,7 @@ var (
 		OOBReportingPeriod:      stringp("0.005s"),
 		BlackoutPeriod:          stringp("0s"),
 		WeightExpirationPeriod:  stringp("60s"),
-		WeightUpdatePeriod:      stringp(".050s"),
+		WeightUpdatePeriod:      stringp("30s"),
```

Comment:
Why is this changed?

Reply:
Not strictly needed; but I figured I'd make it hard for the scheduler to emit during the test which would race with test (but I protect with mu so no issue there).